### DISCUSSION TITLE
How to expand monorepo group?

### DISCUSSION THREAD

**Discussion by Methuselah96** [renovatebot/renovate#11460] [Request Help] [answered]
I want to group all of the packages that require React 17 into one group. This group would include:

- Everything already in the React monorepo.
- Corresponding type packages: `@types/react`, `@types/react-dom`, `@types/react-test-renderer`.
- Packages that require React 17 as a peer dependency: `react-three-fiber`

How can I modify the Renovate config to include these all in the same group (more importantly, the same PR)?

**Comment by rarkins** [comment, marked as answer]
Here's the base monorepo preset: https://docs.renovatebot.com/presets-monorepo/#monoreporeact

And here's how the grouping is done: https://docs.renovatebot.com/presets-group/#groupreactmonorepo

To extend the group you just need them to have the same `groupName`. e.g.

```json
{
  "packageRules": [{
    "matchPackageNames": ["@types/react", "@types/react-dom", "@types/react-test-renderer", "react-three-fiber"],
    "groupName": "react monorepo"
  }]
}
```

**Reply by Methuselah96** [reply]
Thanks, I was adding the `"extends": "monorepo:react"` which didn't work but that did it!
